src/v0/destinations/fb_custom_audience/recordTransform.js

```diff
@@ -100,18 +100,11 @@ const processRecordEventArray = (
 };
 
 async function processRecordInputs(groupedRecordInputs) {
-  const { destination } = groupedRecordInputs[0];
+  const { destination, connection } = groupedRecordInputs[0];
   const { message } = groupedRecordInputs[0];
-  const {
-    isHashRequired,
-    accessToken,
-    disableFormat,
-    type,
-    subType,
-    isRaw,
-    maxUserCount,
-    audienceId,
-  } = destination.Config;
+  const { accessToken, disableFormat, type, subType, isRaw, maxUserCount } = destination.Config;
+  const audienceId = get(connection, 'config.destination.audienceId');
```

Comment:
why do we need to use get here?
what about 
connection?.config?.destination?.audienceId